### PR TITLE
Add GHA to build and push operator and runner images on merge

### PR DESCRIPTION
**Pull request by rooftopcellist**
This will keep the following images up to date:

* quay.io/ansible/awx-resource-operator:devel
* quay.io/ansible/awx-runner-operator:devel

The images will be built and pushed upon merge to devel.

### CODE REVIEW

**Comment by rooftopcellist** [comment]
After merging, check that the job passed.  The Github secret may need to be updated.

File: .github/workflows/build-operator-image.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          password: ${{ secrets.QUAY_TOKEN }}
+  runner-image-build:
+    runs-on: ubuntu-latest
+    name: Push Runner Image
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build Image
+        run: |
+          docker build -t awx-runner-operator:devel -f Dockerfile.runner .
```
Comment (dsavineau) [inline .github/workflows/build-operator-image.yaml]:
```suggestion
          docker build -t awx-resource-runner:devel -f Dockerfile.runner .
```

File: .github/workflows/build-operator-image.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+      - name: Push To Quay
+        uses: redhat-actions/push-to-registry@v2.1.1
+        with:
+          image: awx-resource-operator
```
Comment (dsavineau) [inline .github/workflows/build-operator-image.yaml]:
```suggestion
          image: awx-resource-runner
```

**Review by dsavineau — COMMENTED** [review]
Looks like the runner image reference on quay is wrong

https://quay.io/repository/ansible/awx-resource-runner

**Comment by rooftopcellist** [comment]
right you are!  I just fixed that, thanks!

**Review by dsavineau — APPROVED** [review]

**Pull request merged**